autoload/ctrlp/mrufiles.vim

```diff
@@ -56,16 +56,17 @@ fu! s:reformat(mrufs, ...)
 endf
 
 fu! s:record(bufnr)
-	if s:exclnomod && &l:modifiable | retu | en
-  if s:exclnomod && !&l:modifiable | retu | en
 	if s:locked | retu | en
+
 	let bufnr = a:bufnr + 0
 	let bufname = bufname(bufnr)
-	if bufnr > 0 && !empty(bufname)
-		cal filter(s:mrbs, 'v:val != bufnr')
-		cal insert(s:mrbs, bufnr)
-		cal s:addtomrufs(bufname)
-	en
+
+	if bufnr == 0 || !filereadable(bufname) | retu | en
```

Comment:
Ok, I reestablished everything as before. The exclude_nomod option is gone, only
```vim
         !empty(getbufvar('^' . abs_fn . '$', '&bt')) 
```
left. Check my last push.